docs/community/contributor_ladder.mdx

```diff
@@ -0,0 +1,178 @@
+    * Report and sometimes resolve issues
+    * Occasionally submit PRs
+    * Contribute to the documentation
+    * Show up at meetings, take notes
+    * Answer questions from other community members
+    * Submit feedback on issues and PRs
+    * Test releases and patches and submit reviews
+    * Run or helps run events
+    * Promote the project in public
+    * Help run the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become an Organization Member
+
+### Owners Structure
+There are two types of owners in the ORAS project organized hierarchically. 
+ORAS org owners oversee the overall project and its health. 
+Subproject owners focus on a single repository, 
+a group of related repositories, a service (e.g., a website), 
+or subproject to support the other subprojects (e.g., marketing or community management).
+
+Changes in Oras Org owners have to be announced via an oras-project/community/issues. 
+Changes to sub-project owners are to be announced via the appropriate sub-project issue.
+
+### Owner Responsibility
+ORAS owners adhere to the requirements and responsibilities set forth in the respective ORAS Org Owners and Subproject Owners. 
+They further pledge the following:
+
+* To act in the best interest of the project and subprojects
+* To ensure that project and subproject development and direction is a function of community needs
+* To take action when reasonably confident that it is the right action to take
+* To fulfill the responsibilities outlined in this document and its dependents
+
+### ORAS Org Owners
+The ORAS Org owners are responsible for:
+
+* Maintaining the mission, vision, values, and scope of the project
+* Refining the governance and charter as needed
+* Making project level decisions
+* Resolving escalated project decisions when the subproject owners responsible are blocked
+* Managing the ORAS brand
+* Controlling access to ORAS assets such as source repositories, hosting, project calendars
+* Deciding what subprojects are part of the ORAS project
+* Deciding on the creation of new subprojects
+* Overseeing the resolution and disclosure of security issues
+* Managing financial decisions related to the project
+
+Changes to org owners use the following:
+
+* Any subproject owner is eligible for a position as an org owner.
+* No one company or organization can employ a simple majority of the org owners.
+* An org owner may step down by submitting an issue stating their intent and they will be moved to emeritus.
+* Org owners MUST remain active on the project. If they are unresponsive for > 6 months they will lose org ownership unless a super-majority of the other org owners agrees to extend the period to be greater than 6 months.
+* When there is an opening for a new org owner, any current owners may nominate a suitable subproject owners as a replacement
+    * Nominations for new owners must be made by creating an issue.
```

Comment:
Creation of a new issue in what location?

Reply:
@sabre1041 maybe community?

Reply:
The community repo sounds good